modules/updating-virtual-hardware-on-vsphere.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// updating/updating-hardware-on-nodes-running-in-vsphere.adoc
+
+[id="updating-virtual-hardware-on-vsphere_{context}"]
+= Updating virtual hardware on vSphere
+
+To update the hardware of your virtual machines (VMs) on VMware vSphere, you must power off all of your cluster nodes before upgrading. Once your nodes are updated, you can power them back on.
```

Comment:
We should follow the typical maintenance of OCP, not sure we should power off all the cluster nodes

Reply:
According to comments in [OSDOCS-2629](https://issues.redhat.com/browse/OSDOCS-2620), shutting down all nodes at once is a supported operation. @gnufied any thoughts on this?

Reply:
I asked in openshift-arch and yeah it is supported operation but that does not mean they should do it. It will result in complete cluster outage and hence should not be done

Reply:
Or do we just point to the article Joe referenced in a later comment? https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere-lifecycle-manager.doc/GUID-BE02FDB4-8C09-4EEF-93F0-E0519D19FE70.html